pkg/kms/service.go

```diff
@@ -132,6 +134,26 @@ func (s *service) Decrypt(keystoreID, keyID string, cipher, aad, nonce []byte) (
 	return s.crypto.Decrypt(cipher, aad, nonce, kh)
 }
 
+// ComputeMAC computes message authentication code (MAC) for data.
+func (s *service) ComputeMAC(keystoreID, keyID string, data []byte) ([]byte, error) {
+	kh, err := s.getKeyHandle(keystoreID, keyID)
+	if err != nil {
+		return nil, err
```

Comment:
Please wrap this error

Reply:
I am rewriting error handling here (service) and in operations. In this particular case, `getKeyHandle` is a helper function that already wraps an error.